microsoft-edge/webview2/concepts/security.md

```diff
@@ -11,41 +11,59 @@ ms.date: 10/14/2020
+1. **Design specific web messages and host object interactions, instead of using generic proxies.**
 
-*  Design specific web messages and host object interactions, instead of using generic proxies.
-
-*  Set the following options to restrict web content functionality, by modifying [ICoreWebView2Settings (Win32)](/microsoft-edge/webview2/reference/win32/icorewebview2settings) or [CoreWebView2Settings (.NET)](/dotnet/api/microsoft.web.webview2.core.corewebview2settings):
 
+1. **Restrict Web Content Functionality if not needed**:
+    Update the WebView2 settings in `CoreWebView2Settings` to restrict the functionality of the web content
+   
    *  Set `AreHostObjectsAllowed` to `false`, if you don't expect the web content to access host objects.
 
    *  Set `IsWebMessageEnabled` to `false`, if you don't expect the web content to post web messages to your native application.
 
    *  Set `IsScriptEnabled` to `false`, if you don't expect the web content to run scripts (for example, when showing static HTML content).
 
    *  Set `AreDefaultScriptDialogsEnabled` to `false`, if you don't expect the web content to show `alert` or `prompt` dialogs.
-
-*  Update settings based on the origin of the new page:
+    
+    ##### [.NET/C#](#tab/dotnetcsharp)
+    
+    * [CoreWebView2Settings Class](dotnet/api/microsoft.web.webview2.core.corewebview2settings)
+    
+    ##### [WinRT/C#](#tab/winrtcsharp)
+    
+    * [CoreWebView2Settings Class](microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2settings)
+    
+    ##### [Win32/C++](#tab/win32cpp)
+    
+    * [ICoreWebView2Settings](microsoft-edge/webview2/reference/win32/icorewebview2settings)
+    ---
+            
+
+1. **Update settings based on the origin of the new page:**
 
    *  To prevent your application from navigating to certain pages, use the `NavigationStarting` and `FrameNavigationStarting` events to check page or frame navigation, and then conditionally block the navigation.
 
    *  When navigating to a new page, you may need to adjust the property values on [ICoreWebView2Settings (Win32)](/microsoft-edge/webview2/reference/win32/icorewebview2settings) or [CoreWebView2Settings (.NET)](/dotnet/api/microsoft.web.webview2.core.corewebview2settings), as previously described.
 
-*  When navigating to a new document, use the `ContentLoading` event and `RemoveHostObjectFromScript` to remove exposed host objects.
+1.  When navigating to a new document, use the `ContentLoading` event and `RemoveHostObjectFromScript` to remove exposed host objects.
 
-*  WebView2 cannot be run as a system user.  This restriction blocks scenarios such as building a Credential Provider.
+1. WebView2 cannot be run as a system user.  This restriction blocks scenarios such as building a Credential Provider.
 
 
 <!-- ====================================================================== -->
 <!--
 ## Security
 
+TODO: should we include this section?
```

Comment:
@vbryh-msft @LiangTheDev I found the following best practices hidden behind a comment block in this document. Should they be included?

Reply:
Do we know who added these statements originally? That person might have more context on why they are commented out.

Reply:
strange to have a "Security" h2 section within the "Develop secure WebView2 apps" article.  What's this hidden "Security" section about, specifically?  change h2 to that

Reply:
in file tree in "main", for file:
https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/webview2/concepts/security.md
click History in upper right, to see who pushed commits to this .md file

Reply:
Based on Git blame, it's a person called Joe Martin https://github.com/jm-trd-ms that wrote this comment.
Any idea on this @mikehoffms ?

Reply:
@victorhuangwq - J.M. is no longer on this team

Reply:
I would like to figure out if these best practices are still valid - if so I am inclined to edit them and add them to the list. If we delete them, we likely won't add them back again.

@david-risney Since you know a lot about interoperability, I am wondering if you know if these best practices that was written by JM is still applicable to the current webview2?

Reply:
that whole section is currently commented-out.

Rendered article (the visible sections):
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/security?branch=pr-en-us-3195